docs/editor/integrated-terminal.md

```diff
@@ -164,6 +164,12 @@ The basics of the terminal have been covered in this document, read on to find o
 
 ```bat
 @echo off
+SET ARGS=%*
+IF "%ARGS:~0,5%" EQU "/d /c" ( 
+    REM Tasks call the terminal expecting it runs and exits
+    %ARGS:~6%
```

Comment:
In this new commit, I'm calling the desired Task command directly in the terminal, removing the _"/d /c"_ flags that are not applicable in here.